README.md

```diff
@@ -35,7 +39,9 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+var andy = Instructor.findOne({name: 'Andy'}).then(result => {
+  WishlistItems.create({instructor: result, description: 'Resin Laying Deer Figurine, Gold'})
+})
```

Comment:
You should push the new wishlist item into the instructors `wishlistitems` array and then save that instructor.